Add Client and Audience sections to the Configuration-Authentication topic. (#391)
Mirantis/mke-docs: content/docs/configuration/authentication/_index.md

| `expiry.refreshTokens.disableRotation`   | Disable every-request rotation.                                          |
| `expiry.refreshTokens.reuseInterval`     | Interval for obtaining the same refresh token from the refresh endpoint. |

> ## Add Third-Party Client applications

Reviewer: ```suggestion
## Add Third-Party Client Applications
```

You can use third-party client applications with MKE 4k, each with the ability
to use the same authentication options that are available in the MKE 4k

> cluster. To do this, you obtain the current MKE4.yaml configuration file, add a

Reviewer: ```suggestion
cluster. To do this, obtain the current `MKE4.yaml` configuration file, add a
```

list of the client applications to the file, and reapply the file to the MKE 4k
cluster.

Example configuration:

> ```

Reviewer: ```suggestion
```yaml
```

authentication:
    clients:
    - id: example
      name:"Example Client"
      redirectURIs:
      - localhost:8080
      secret: ZXhhbXBsZS1hcHAtc2VjcmV0
```

| Field                                    | Description                                                              |
| ---------------------------------------- | ------------------------------------------------------------------------ |
| `clients`                                 | Section for the various client settings, within which each client is represented as a list.                                 |
| `clients.id`                                 | Unique ID for the client, which must match the ID passed by the client during authentication.                                 |
| `clients.name`                                 | Human-readable name for the client.                                 |
| `clients.redirectURIs`                                 | A list of URIs for the client from which an authentication will accept auth requests and to which an authentication will return successful auths to.                                 |

> | `clients.secret`                                 | A `secret` that is passed by the client to validate that it is allowed to use authentication. Its intended use is with back end client applications that can keep the secret hidden.                                 |

Reviewer: ```suggestion
| `clients.secret`                                 | A `secret` provided by the client to validate that it is allowed to use authentication. Its intended use is with backend client applications that can keep the secret hidden.                                 |
```

Author: Keeping 'passed by' as it is developer-ese that is in common use. Also, the client is more of a conduit for the secrte than the provider of such.

| `clients.public`                                 | Eliminates the need for a secret that must be shared with each client setup.                                 |

## Add Audiences

> Audiences determine which parties that are permitted to make requests to the

Reviewer: ```suggestion
Audiences determine which parties are permitted to make requests to the
```

cluster API. Pairing an audience with a client application enables those

> clients that display or change information about the cluster.

Reviewer: ```suggestion
clients to display or change information about the cluster.
```

Author: Resolving the sentence as follows, following discussion on Slack:

"Adding the client as an audience allows it to leverage the Kubernetes API to display and edit information about the cluster."

> ```

Reviewer: ```suggestion
```yaml
```
